src/maliput_malidrive/test_utilities/assert_compare.h

```diff
@@ -0,0 +1,46 @@
+// BSD 3-Clause License
```

Comment:
What if we move it to the test folder so there is no doubt about this file being used in production libraries?

Reply:
test_utilities  (in malidrive) is not part of the public API

Reply:
Moved to test at d154041